about_us.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: default
+title: About Us
+nav_order: 10
+---
+
+# About Us
+{: .no_toc }
+## Table of contents
+{: .no_toc .text-delta }
+
+1. TOC
+{:toc}
+
+<img src="../images/medik8s-logo.png" alt="nmo-icon" width="200" style="margin-left:auto; margin-right:auto; display:block"/>
+
+## Who are we?
+
+We, Medik8s team, are four Red Hatters with a strong passion to High Availabitlity (HA) solutions in the Kubernetes world.
+
+## What we do?
+
+We develop Kubernetes Open Source Operators using [Operator Lifecycle Manager (OLM)](https://olm.operatorframework.io/) that provide maintenance support, automatic node remediation and high availability for singleton workloads:
+
+- [Node Healthcheck Operator (NHC)](failure_detection) - Detecting Node Failures, triggering remediation, which is performed by other operators like Self Node Remediation or MachineDeletion.
+- [Self Node Remediation (SNR)](/remediation/self-node-remediation/self-node-remediation) - Remediates nodes by rebooting, without needing a management interface like e.g. IPMI, or a node / machine provisioning API. Works standalone, and / or with NHC.
+- [Node Maintenance Operator (NMO)](maintenance-node) - Declarative node cordoning and draining prior to harmful decisions.
+
+## What we plan to do?
+
+We plan to improve the exiting operators by adding Master (control plane) fencing capabilites to SNR and NHC, which currently **only** work for worker nodes.
+Furthermore, we have few work in progress operators:
+
+- [Machine Deletion](/remediation/machine-deletion/machine-deletion) - Remediates nodes by deleting the associated OpenShift machine. Triggered by NHC.
+- [HA-SNO](https://github.com/medik8s/ha-sno) - High Availability Single Node Openshift.
+
+## History
+
+In 2018, the team behind medik8s prototyped what would eventually ship as the
+Machine Healthcheck Controller for OpenShift 4.2.
+
+Soon after, Red Hat brought the Machine Healthcheck Controller to sig-cluster
+for consideration as a general purpose mechanism for detecting node failures and
+recovering compute power and affected workloads.
+
+In 2019, we improved support for bare metal by shipping an
+annotation based mechanism for rebooting nodes instead of going through a time
+expensive reprovisioning cycle.
+
+In 2020, we worked with Ericsson to design an official API for using alternative
+mechanisms to recover bad nodes.  Since then Ericsson has prototyped a
+[metal3](http://metal3.io/) based implementation, and we have implemented
+[Self Node Remediation](https://github.com/medik8s/self-node-remediation) for shared-nothing environments.
+
+In 2021, we created medik8s to make general purpose HA available
+to all kubernetes clusters, not just ones backed by an infrastructure API.
+
+
+## Community
+Join [our google group](https://groups.google.com/g/medik8s) to get more info, participate in dicussions and get notified
```

Comment:
Maybe put first our public slack ? (I think it's probably used more)

Reply:
It is indeed more used, but it is only for Red Hatters...